br/pkg/stream/rewrite_meta_rawkv.go

```diff
@@ -261,10 +287,15 @@ func (sr *SchemasReplace) rewriteTableInfo(value []byte, dbID int64) ([]byte, bo
 
 	tableReplace, exist := dbReplace.TableMap[tableInfo.ID]
 	if !exist {
-		newID, err := sr.genGenGlobalID(context.Background())
-		if err != nil {
-			return nil, false, errors.Trace(err)
+		newID, exist = sr.globalTableIdMap[tableInfo.ID]
+		if !exist {
+			newID, err = sr.genGenGlobalID(context.Background())
```

Comment:
If we don't want to adapt context for now, I think `context.TODO()` would be better.
```suggestion
			newID, err = sr.genGenGlobalID(context.TODO())
```